{{cookiecutter.project_slug}}/_/development/python/docker-compose.yml

```diff
@@ -27,6 +27,9 @@ services:
 {%- if cookiecutter.database == 'Postgres' %}
   database:
     image: docker.io/bitnami/postgresql:12
+    environment:
+      POSTGRESQL_PASSWORD: postgres
```

Comment:
I believe, this one is already [covered by the defaults](https://github.com/painless-software/painless-continuous-delivery/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/_/frameworks/Django/application/settings.py#L104) in the Django settings. We could omit it without any damage. Can you try?

Reply:
OK, I just tried it.  Unfortunately, it does need it.  It needs both or it doesn't work. :(

Reply:
Interesting. Looks like, unlike the Postgres image we used before, the Bitnami image requires to explicitly set a password for their default `postgres` user in the default `postgres` database. Makes sense.

So, we have to live with having to specify at least the password explicitly.